src/components/Blockquote.astro

```diff
@@ -2,7 +2,7 @@
 import { Icon } from "astro-icon"
 ---
 
-<figure class="relative border-y-[3px] border-astro-gray-500 py-4">
+<figure class="relative my-8 border-y-[3px] border-astro-gray-500 py-4">
```

Comment:
Do we use this component anywhere else yet?

I usually try to keep spacing between elements in the container itself, but if we always want a top margin here it might be a great case for `margin-trim` to ignore it if this is the first child

Reply:
Yeah that's a good point, I'll figure out some other way to accomplish this

Reply:
I looked, and this PR is the first usage of this component 😅 I guess I'll just keep it for now?